nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -206,6 +206,11 @@ language governing permissions and limitations under the License. -->
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.xerial.snappy</groupId>
+            <artifactId>snappy-java</artifactId>
```

Comment:
So unfortunately I think this library may be a no-go.  In reviewing the LICENSE it is said to be ASLv2.  However, in reviewing their NOTICE as-is necessary to copy forward any appropriate NOTICE data there is a GCC Exception (GPL v3).  https://github.com/xerial/snappy-java/blob/develop/NOTICE  I do not believe this is allowed under ASF licensing guidelines.  More on the topic of even GPLv3 w/Exceptions being not allowed can be found here http://www.apache.org/legal/resolved.html